src/medication-request/medication-request.resource.tsx

```diff
@@ -27,6 +27,8 @@ import {
   OPENMRS_FHIR_EXT_REQUEST_FULFILLER_STATUS,
 } from "../constants";
 
+export const REFRESH_INTERVAL = 10000; // 10 second refresh interval
```

Comment:
Seems fine.  If easy, could also pull this from config with a default of 10000, which would allow changing this if after testing a different value is desired.